db/migrate/20210105165605_add_processed_to_notifications.rb

```diff
@@ -2,13 +2,16 @@
 
 class AddProcessedToNotifications < ActiveRecord::Migration[6.0]
   def up
-    add_column :notifications, :processed, :boolean, default: false
-    execute "UPDATE notifications SET processed = true"
-    change_column_null(:notifications, :processed, false)
-    add_index :notifications, [:processed], unique: false
+    # Commented out because this was causing issues with large databases.
```

Comment:
I think it's fine to remove the comments. It will always be in the git history.

Reply:
You would prefer this file be completely blank?

Reply:
great question! I'll try to see.